scaffolding/src/commands.rs

```diff
@@ -32,7 +30,10 @@ impl CommandApplicator {
             match self.rx.try_recv() {
                 Ok(cmd)             => {
                     redraw = true;
-                    cmd.apply(&mut terminal).unwrap_or_else(exit_on_io_error);
+                    cmd.apply(&mut terminal).unwrap_or_else(|e| {
+                        panic!("Error applying command '{}' due to error: {}",
+                               cmd.repr(), e);
```

Comment:
@withoutboats Just to be clear, before I add the `gtk::main_quit()` call back into this arm of the match, I would like more information from you about the error you are seeing. I don't necessarily think that blindly shutting down the gtk main loop on any error is the right thing to do here so I would like to handle that case specifically.